File: BookMyShow.Web/bookMyShow/src/app/shared/models/reserved-seat.ts
import { Seat } from "./seat";

export class ReservedSeat extends Seat {
    theatreId: number;
    movieId: number;
    showTime: string;
    constructor(isReserved: boolean, isDisabled: boolean, seatNumber: string, movieId: number,
        showTime: string, theatreId: number) {
        super(seatNumber, isReserved, isDisabled)
        this.theatreId = theatreId;
        this.showTime = showTime;
        this.seatNumber = seatNumber;
        this.movieId = movieId
    }
}

File: BookMyShow.API/BookMyShow.DomainModels/ReservedSeat.cs
namespace BookMyShow.DomainModels
{
    public class ReservedSeat
    {
        public string SeatNumber { get; set; }
        public int TheatreId { get; set; }
        public int MovieId { get; set; }
        public string ShowTime { get; set; }
    }
}
